### DISCUSSION TITLE
Port 443 no permission

### DISCUSSION THREAD

**Discussion by krowisoft** [nextcloud/all-in-one#3990] [Questions] [answered]
### Steps to reproduce
1.Copy command to set up the container
2.change port from 8080 to 8085 because of another container on the system
3.run command

### Expected behavior 
Start with automatically opening port 443
### Actual behavior 
Somehow the permission is getting denied. I tried to give the container the port 443 tcp but it already was configured with port 443 allowed. 
2024-01-01 20:29:11: (../src/network.c.588) bind() [::]:443: Permission denied is what was logged inside the container.

### Host OS 
Ubuntu 20.04.6 LTS x86_64

#### Nextcloud AIO version 
Nextcloud AIO v7.9.0

#### Other valuable info 
I cant see the channel

**Comment by krowisoft** [comment]
Even when not changing the port from 8080 to 8085 I get the same error

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1267#discussioncomment-3885993
